### PR TITLE
Feature/learn about the infrastructure

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/infrastructure/node-types.md [hunk 1]
```diff
@@ -16,18 +18,37 @@ Mix nodes anonymously relay encrypted Sphinx packets between each other, adding
...
 In addition, gateways also cache messages, functioning as an inbox for users who are offline. By providing secure, reliable access to the mixnet and ensuring that data remains protected, gateways play a crucial role in maintaining the integrity of the Nym network.
 
-### Validators
+#### Validators
 
 Validators are essential to the security and integrity of the Nym network, tasked with several key responsibilities. They utilize proof-of-stake Sybil defense measures to secure the network and determine which nodes are included within it. Through their collaborative efforts, validators create Coconut threshold credentials which provide anonymous access to network data and resources.
 
 Validators also play a critical role in maintaining the Nym Cosmos blockchain, a secure, public ledger that records network-wide information such as node public information and keys, network configuration parameters, CosmWasm smart contracts, and `NYM` and credential transactions.
 
+#### Service Provider
```
Comment (mfahampshire) [inline src/infrastructure/node-types.md:35]: typo - should be `providerS`

File: src/infrastructure/node-types.md [hunk 2]
```diff
@@ -16,18 +18,37 @@ Mix nodes anonymously relay encrypted Sphinx packets between each other, adding
...
+
+Service Providers are a crucial aspect of the Nym infrastructure that support the application layer of the Nym network. Any application built with Nym will require a Service Provider, which can be created by anyone. TService Providers run a piece of binary code that enables them to handle requests from Nym users or other services, and then make requests to external servers on behalf of the users.
```
Comment (mfahampshire) [inline src/infrastructure/node-types.md]: Typo @ start of the 2nd sentence

File: src/infrastructure/node-types.md [hunk 3]
```diff
@@ -16,18 +18,37 @@ Mix nodes anonymously relay encrypted Sphinx packets between each other, adding
...
+
+For example, a Service Provider could receive a request to check a mail server and then forward the response to the user. Importantly, Service Providers are not open proxies and have a domain whitelist, which ensures that they only communicate with trusted domains.
```
Comment (mfahampshire) [inline src/infrastructure/node-types.md]: Conflating Network Requester and Service Providers here - clarify

File: src/infrastructure/node-types.md [hunk 4]
```diff
@@ -16,18 +18,37 @@ Mix nodes anonymously relay encrypted Sphinx packets between each other, adding
...
 
+#### Service Provider
+
+Service Providers are a crucial aspect of the Nym infrastructure that support the application layer of the Nym network. Any application built with Nym will require a Service Provider, which can be created by anyone. TService Providers run a piece of binary code that enables them to handle requests from Nym users or other services, and then make requests to external servers on behalf of the users.
+
+For example, a Service Provider could receive a request to check a mail server and then forward the response to the user. Importantly, Service Providers are not open proxies and have a domain whitelist, which ensures that they only communicate with trusted domains. 
+
+Service Providers earn credentials by facilitating the flow of information in and out of the mixnet, and these credentials can be redeemed for `NYM` tokens as a reward for providing services. The presence of Service Providers in the Nym network enhances its security and privacy, making it a reliable and robust platform for anonymous communication and data exchange.
```
Comment (mfahampshire) [inline src/infrastructure/node-types.md]: Not yet they can't - either make this clear or remove

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/infrastructure/implementing-nym.md [hunk 5]
```diff
@@ -31,23 +31,23 @@ The Nym Websocket client is a software tool that allows for communication betwee
...
 
 This client is useful for allowing existing applications to use the Nym mixnet without any code changes. All that's necessary is that they can use the SOCKS5 proxy protocol (which many applications can - crypto wallets, browsers, chat applications etc). It's less flexible as a way of writing custom applications than the other clients.
 
-For more information and guidance on how to implement the Nym SOCKS5 client, please visit [here](https://nymtech.net/docs/stable/integrations/socks5-client).
+For more information and guidance on how to implement the Nym SOCKS5 client, please visit [here](https://nymte.ch/docs/clients/socks5-client.html).
```
Comment (mfahampshire) [inline src/infrastructure/implementing-nym.md]: change staging domain to prod please

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/infrastructure/node-types.md (hunk 1 above)
Comment (oliver-anyanwu-nym) [reply src/infrastructure/node-types.md:35]: Provider* ??

Comment (oliver-anyanwu-nym) [reply src/infrastructure/node-types.md:35]: sorry don't get that:
1) Service Providers
2) Providers
3) Service providers


which one?

File: src/infrastructure/node-types.md [hunk 6]
```diff
@@ -1,7 +1,9 @@
-# Different Types of Nodes
+# Node Types
 
 Discover the intricate workings of Nym's privacy-enhancing mixnet infrastructure through our informative video, where we break down the different types of nodes and how they each play a crucial role in ensuring secure and anonymous communication.
 
+For a more in-depth understanding of the inner workings of Nym's mixnets, check out this on the [Mixnet Traffic Flow](https://nymte.ch/docs/architecture/traffic-flow.html).
```
Comment (mfahampshire) [inline src/infrastructure/node-types.md]: we only have 1 mixnet..

File: src/infrastructure/implementing-nym.md [hunk 7]
```diff
@@ -41,13 +41,13 @@ This client is useful for allowing existing applications to use the Nym mixnet w
...
 
 If you're working in JavaScript, or building an [edge computing app](https://en.wikipedia.org/wiki/Edge_computing), you'll likely want to choose the Webassembly client. We expect that many client apps will be built using the Webassembly client. It's packaged and [available on the npm registry](https://www.npmjs.com/login?next=%2Fpackage%2F%40nymproject%2Fnym-client-wasm), so you can `npm install it` into your JavaScript or TypeScript application.
 
-For more information and guidance on how to implement the Nym Webassembly client, please visit [here](https://nymtech.net/docs/stable/integrations/wasm-client).
+For more information and guidance on how to implement the Nym Webassembly client, please visit [here](https://nymte.ch/docs/clients/webassembly-client.html).
```
Comment (mfahampshire) [inline src/infrastructure/implementing-nym.md]: still got staging links in here - change please

File: src/infrastructure/implementing-nym.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ The Nym Websocket client is a software tool that allows for communication betwee
 
 This option is the native websocket client (`nym-client`). This is a compiled program that can run on Linux, Mac OS X, and Windows machines. It runs as a persistent process on a desktop or server machine. You can connect to it from any language that supports websockets.
 
-For more information and guidance on how to implement the Nym Websocket client, please visit [here](https://nymtech.net/docs/stable/integrations/websocket-client).
+For more information and guidance on how to implement the Nym Websocket client, please visit [here](https://nymte.ch/docs/clients/websocket-client.html).
```
Comment (mfahampshire) [inline src/infrastructure/implementing-nym.md]: staging link

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/infrastructure/nym-vs-others.md [hunk 9]
```diff
@@ -6,6 +6,8 @@ Nym is the first system we're aware of which provides integrated protection on b
 
 Unlike other systems, Nym focuses on __ALL__ layers for privacy and security, providing a more comprehensive solution compared to solely concentrating on one layer like most comparable systems do.
 
+For more information on how Nym differs from other systems, check out this page on the [Mixnet Traffic Flow](https://nymtech.net/docs/stable/architecture/traffic-flow)
```
Comment (mfahampshire) [inline src/infrastructure/nym-vs-others.md]: Missing context and also missing full stop at end of sentence

**Review by mfahampshire — DISMISSED** [review]

**Pull request merged**